packages/metrics/src/aggregate.ts

```diff
@@ -0,0 +1,35 @@
+import { MutationTestResult, TestFileDefinitionDictionary } from 'mutation-testing-report-schema';
+import { determineCommonBasePath, isNotNullish, normalizeFileNames } from './helpers';
+
+/**
+ * Aggregates multiple reports together into a single report, grouped by module.
+ *
+ * @param resultsByModule The MutationTestResult objects by module name
+ * @returns An aggregated result of all provided reports.
+ */
+export function aggregateResultsByModule(resultsByModule: Record<string, MutationTestResult>): MutationTestResult {
+  const projectRoots = Object.values(resultsByModule)
+    .map((report) => report.projectRoot)
+    .filter(isNotNullish);
+  const aggregatedResult: MutationTestResult = {
+    files: {},
+    schemaVersion: '1.7',
```

Comment:
I don't really know, but since we're also merging test views 🤷‍♀️

Reply:
It still works if you don't have a test view, right?

Reply:
Yes. So what would you suggest? 

We could also use `semver.max(...)` or equivalent.

A v1 schema also complies to v1.7, but not the other way around (ok, it might pass validation because backward compatible, but you get the point), that's why I chose that originally

Reply:
Ah, that makes sense. I'm just a little worried about problems a difference might cause. For example, validating a JSON that has `1.7` against the schema which has `1`. It also seems odd to me to only have this code-path result in a different schema version, while it doesn't create any JSON elements that weren't possible in e.g. `1.6`. 

Perhaps `semver.max` would be the best? But then we'd have to add it as a dependency, which might be a bit much. I guess since the schema is backwards compatible, `1` is still true 🤷‍♀️

Reply:
Semver won't work:

```
$ node -p "require('semver').lt('1.1', '1.2')"
C:\z\github\stryker-mutator\mutation-testing-elements\node_modules\semver\semver.js:332
    throw new TypeError('Invalid Version: ' + version)
    ^

TypeError: Invalid Version: 1.1
```

So I would personally like to keep it at 1.7. When we add more fields from later versions, we can up the version. I think it makes sense to keep this field updated.

Our JSON schema only validates that you're using `1.x`: `"pattern": "^1(\\.\\d*)?$"`, so it shouldn't pose an issue 🤷‍♀️